examples/send-and-receive-emails/send-and-receive-emails.description

```diff
@@ -0,0 +1,2 @@
+// The Email Connector is used to do email sending (with SMTP) and receiving (with POP3 or IMAP4) over SSL or STARTTLS protocols.
+// This sample includes sending and receiving emails over SSL with client objects.
```

Comment:
```suggestion
// This sample includes sending and receiving emails using SSL with client objects.
```

Reply:
This is correct as our protocol is running in application layer on top of SSL which runs in transport layer.